internal/services/network/virtual_hub_resource.go

```diff
@@ -78,6 +78,20 @@ func resourceVirtualHub() *pluginsdk.Resource {
 				ValidateFunc: azure.ValidateResourceID,
 			},
 
+			"virtual_router_asn": {
+				Type:     pluginsdk.TypeInt,
+				Computed: true,
+			},
+
+			"virtual_router_ips": {
+				Type:     pluginsdk.TypeSet,
+				Computed: true,
+				Elem: &pluginsdk.Schema{
+					Type: pluginsdk.TypeString,
+				},
+				Set: pluginsdk.HashString,
+			},
```

Comment:
Thank you for the review. Changed it to list as well.

The idea behind "sets" was to prevent configuration changes in the depending NVA appliances (routers) if Azure will send "virtual_router_ips" in a different order. Change of the order will now cause a small connectivity outage.